components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/Property.java

```diff
@@ -23,7 +23,7 @@
 import java.io.Serializable;
 import java.util.Iterator;
 
-public class Property implements Serializable {
+public class Property implements Serializable, Comparable {
```

Comment:
Let's use comparator and sort this in the UI. Doesn't make sense to say a Property is comparable with another.